mkdocs.yml

```diff
@@ -52,84 +83,86 @@ extra:
       link: https://groups.google.com/group/janusgraph-dev
 
 markdown_extensions:
-    - pymdownx.superfences
-    - pymdownx.tabbed
-    - pymdownx.snippets
-    - pymdownx.details
-    - markdown.extensions.admonition
-    - markdown_include.include:
-        base_path: docs
-    - markdown.extensions.codehilite:
-          linenums: True
-    - codehilite
+  - pymdownx.superfences
+  - pymdownx.tabbed
+  - pymdownx.snippets
+  - pymdownx.details
+  - markdown.extensions.admonition
+  - markdown_include.include:
+      base_path: docs
+  - markdown.extensions.codehilite:
+      linenums: True
+  - codehilite
 
 nav:
-    - Introduction: index.md
-    - Getting Started:
-        - Installation: getting-started/installation.md
-        - Basic Usage: getting-started/basic-usage.md
-        - Architectural Overview: getting-started/architecture.md
-    - JanusGraph Basics:
-        - Configuration: basics/configuration.md
-        - Schema and Data Modeling: basics/schema.md
-        - Gremlin Query Language: basics/gremlin.md
-        - JanusGraph Server: basics/server.md
-        - Deployment Scenarios: basics/deployment.md
-        - ConfiguredGraphFactory: basics/configured-graph-factory.md
-        - Things to Consider in a Multi-Node JanusGraph Cluster: basics/multi-node.md
-        - Transactions: basics/transactions.md
-        - JanusGraph Cache: basics/cache.md
-        - Transaction Log: basics/transaction-log.md
-        - Configuration Reference: basics/configuration-reference.md
-        - Config Example: basics/example-config.md
-        - Common Questions: basics/common-questions.md
-        - Technical Limitations: basics/technical-limitations.md
+  - Introduction: index.md
+  - Getting Started:
```

Comment:
I would like this part to be more or less self contained and ideally explain everything to start a JanusGraph Server instance and play with it a bit. That's why I would move the Gremlin chapter into this part and I think I would also split the _JanusGraph Server_ chapter in the _Operations_ part into two chapters, one that explains how to get started with the pre-packaged distributions and one with the more advanced and optional configuration options. Then we could move the first chapter into the _Getting Started_ area and leave only the second chapter in _Operations_.
But we could also do that in a follow-up PR.

Reply:
major text changes should be done in next PR.

Reply:
https://github.com/JanusGraph/janusgraph/issues/1824

Reply:
OK, I just copied my comment over there.